app/controllers/interviews_controller.rb

```diff
@@ -43,8 +53,19 @@ def destroy
 
   private
 
+  def no_conflicts(members, new_interview)
```

Comment:
this should not be in controller, rather in model

Reply:
Can you elaborate the reason for this?

Reply:
Resolved in https://github.com/viditjain5598/BetterInterviews/pull/4